src/modules/cmdpal/Microsoft.CmdPal.UI/Controls/AdaptiveCardsConfig.cs

```diff
@@ -0,0 +1,342 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using AdaptiveCards.Rendering.WinUI3;
+
+namespace Microsoft.CmdPal.UI.Controls;
+
+public sealed class AdaptiveCardsConfig
+{
+    public static AdaptiveHostConfig Light { get; }
+
+    public static AdaptiveHostConfig Dark { get; }
+
+    static AdaptiveCardsConfig()
+    {
+        Light = AdaptiveHostConfig.FromJsonString(LightHostConfigString).HostConfig;
+        Dark = AdaptiveHostConfig.FromJsonString(DarkHostConfigString).HostConfig;
+    }
+
+    public static readonly string DarkHostConfigString = """
```

Comment:
Should we abstract these to files we can load? (Either from the app package to start, if not let them be user configurable later?)

Reply:
They could be, and DevHome did have these as separate files. But I didn't think they _needed_ to be user configurable. And for Terminal, we found there was a marginal startup performance benefit gai if we compiled JSON like this straight into the app, so that we didn't need to hit the filesystem just to read a file that's going to be static from the moment the app is compiled.